README.md

```diff
@@ -19,7 +19,7 @@ Requirements
 Role Variables
 --------------
 
-Variables are used to tune the Rails application that runs Discourse. These are in a template, and set in the `defaults/main.yml` and `vars/main.yml` file. Please read them and set them accordingly in `vars/main.yml`
+Variables are used to tune the Rails application that runs Discourse. These are in a template, and set in the `defaults/main.yml` file. Please read them and set them accordingly in your playbook
```

Comment:
Following our discussion in #14 I suggest we keep the original text, but remove `vars/main.yml` from change control.